docs/api/webviews.md

```diff
@@ -1,373 +1,130 @@
-
-```js
-const innerElement = web.element(by.web.id('inner_element_identifier'))
+```javascript
+const innerElement = web.element(by.web.id('inner_element_identifier'));
 await expect(innerElement).toHaveText('Hello World!');
 ```
 
-The code above finds an inner element by HTML `id` attribute, and expects it to have a specific text.
+This example demonstrates locating an element by its HTML `id` attribute and verifying its text content.
 
-### `web(nativeMatcher).element(matcher)`
+### Multiple Web Views Scenario
 
-If you have multiple web views on the screen, you must locate a specific web view first by using a [native matcher][native matchers], e.g.:
+For screens with multiple web views, first identify the specific web view using a **[native matcher]**. Then, locate the element within that web view.
 
-```js
+```javascript
 const myWebView = web(by.id('webview_identifier'));
-const innerElement = myWebView.element(by.web.id('inner_element_identifier'))
+const innerElement = myWebView.element(by.web.id('inner_element_identifier'));
 await expect(innerElement).toHaveText('Hello World!');
 ```
 
-In the example above:
-
-1. We use `web()` function and [`by.id()`] matcher to locate a web view by its accessibility identifier.
-1. We use `myWebView.element()` method and [`by.web.id()`] web matcher to find an HTML element inside that web view.
-1. We run the same expectation (to have text) as in the previous example.
-
-## Matchers
+#### Using `atIndex` (iOS only)
 
-Web view matchers are used to find elements within a web view:
+It is also possible to locate the web view by applying at-index to the web view matcher:
 
-- [`by.web.id()`]
-- [`by.web.className()`]
-- [`by.web.cssSelector()`]
-- [`by.web.name()`]
-- [`by.web.xpath()`]
-- [`by.web.href()`]
-- [`by.web.hrefContains()`]
-- [`by.web.tag()`]
-- [`atIndex()`]
-
-### `by.web.id(id)`
-
-Match elements with the specified accessibility identifier.
-
-```js
-web.element(by.web.id('identifier'));
+```javascript
+const myWebView = web(by.id('webview_identifier').atIndex(1));
 ```
 
-### `by.web.className(className)`
+In this example:
 
-Match elements with the specified CSS class name.
-
-```js
-web.element(by.web.className('className'));
-```
-
-### `by.web.cssSelector(cssSelector)`
-
-Match elements with the specified CSS selector.
-
-```js
-web.element(by.web.cssSelector('#cssSelector'));
-```
+1. The `web()` function and `by.id()` matcher locate the web view by its accessibility identifier.
+2. The `myWebView.element()` method and `by.web.id()` matcher locate an HTML element within the web view.
+3. The expectation to verify the element's text is the same as in the single web view scenario.
 
-### `by.web.name(name)`
```

Comment:
Why did you so brutally cut down the article? I don't understand.

Reply:
Agreed to revert to the old document (with some changes) and create a guide based on the new document (which is more of a guide than an API document)